doc/source/ray-observability/monitoring-debugging/troubleshoot-failures.rst

```diff
@@ -1,44 +1,252 @@
 Troubleshooting Failures
 ========================
 
-Crashes
--------
+What Kind of Failures Exist in Ray?
+-----------------------------------
 
-If Ray crashed, you want to know why and what happened. Currently, this can be attributed to the following reasons:
+Ray consists of 2 major APIs. :func:`ray.remote <ray.remote>` to create a task/actor and :func:`ray.get <ray.get>` to get the result. 
```

Comment:
it's not ray.remote that creates a task/actor, it only defines one. It should be RemoteFunction.remote()

Reply:
I will just remove the reference here. That reference is really confusing. I will just mention`.remote`. I believe everyone who uses ray should know that that means